### ISSUE TITLE
App not loading on M365 android app, no changes were done on tab or app

### ISSUE THREAD

**Issue by ktcodebite** [MicrosoftDocs/msteams-docs#8235]
Asking on behalf of partner Jira, tab is not loading on Android M365 app without making any changes to the app. Please reach out to kithoma@microsoft.com for more details on the problem

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite - Thanks for reporting your issue. Could you please share the error which you are getting while loading the tab?
Are you trying to add static tab or configurable tab?

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite - We have followed steps which is mentioned in below document for Installing Microsoft 365 apps in test environment
https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/prerequisites#mobile

Tested with below manifest. The static tab is loading fine at our end.
[manifest.zip](https://github.com/MicrosoftDocs/msteams-docs/files/10880285/manifest.zip)

![image](https://user-images.githubusercontent.com/92150070/222678362-aaf1b7dd-342e-4f88-a61a-c2b48c99a1d1.png)

![image](https://user-images.githubusercontent.com/92150070/222678480-552b1eb5-caac-4a3f-ac8f-2b041e2e2237.png)

We have tested this on Android 12 and Microsoft 365 version 1.0.1(16.0.16227.20032) by joining the beta program.

Could you please follow the document for installing Microsoft 365 apps in test environment and check again.

**Comment by vbatr** [comment]
Hi @Nivedipa-MSFT  

the content context is working without any issues, but [dialog.open()](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fjavascript%2Fapi%2F%40microsoft%2Fteams-js%2Fdialog.url%3Fview%3Dmsteams-client-js-latest%23%40microsoft-teams-js-dialog-url-open&data=05%7C01%7CKiran.Thomas%40microsoft.com%7Ccbae8e6494934b8077fc08db1b43a2b9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638133752360798187%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000%7C%7C%7C&sdata=mGCIXIz%2BtH93Dppapd6yijWFSlAZ5XJCpoPv%2BPe9rmQ%3D&reserved=0) called from the tab is not working at all, and task module is not opened...

**Comment by Wajeed-msft** [comment]
@vbatr - Is this duplicate of issue https://github.com/MicrosoftDocs/msteams-docs/issues/8234?

**Comment by vbatr** [comment]
@Wajeed-msft nope, that is two separate issues. #8234 is failing on trying to get auth token (await microsoftTeams.authentication.getAuthToken({ silent: true }) ) in Outlook desktop client for task modules. But if we do an upgrade to 2.8.0+ version of the library we have another issue with the same symptoms like in current case


#8235 is related to dialog.open() on Microsoft 365 Androind app (for the lib v 2.6.0 +)

**Comment by Wajeed-msft** [comment]
Thanks for clarification. 
@Nivedipa-MSFT - could you please test **dialog.open()** on Microsoft 365 **Android app** using SDK **v2.6.0**?

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite - We have tested dialog.open() on Android 12 and Microsoft 365 version 1.0.1(16.0.16227.20032) by joining the beta program. using SDK v2.6.0. 

https://user-images.githubusercontent.com/92150070/222714315-2e6cb2b0-0c4f-4e62-9e49-2af4e9a626cd.mp4


We are able to repro your issue. We have raised a bug for the same. We will let you know once we have any updates on it.

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite - We got an update from engineering team saying that dailog.open() is not currently supported in Android mobile.
